x-pack/legacy/plugins/code/model/search.ts

```diff
@@ -20,17 +20,49 @@ export interface Document {
   sha1?: string;
 }
 
+export interface Commit {
+  id: string;
+  message: string;
+  body: string;
+  date: Date;
+  parents: string[] | string;
```

Comment:
is it simpler to just make it `string []` instead of `string[] | string`

Reply:
done